Gitter/Gitter/Gitter.WindowsPhone/Views/AboutPage.xaml

```diff
@@ -92,16 +95,32 @@
                                     <ColumnDefinition Width="60" />
                                     <ColumnDefinition />
                                 </Grid.ColumnDefinitions>
-                                
-                                <Image Grid.Column="0" 
-                                       Source="{Binding Path=Image}" />
+
+                                <Grid Grid.Column="0">
```

Comment:
Ah gotcha, okay that works, and I can now see that this is the same on the ChatMessageControl, so it all makes sense now.  I have pushed the changes, so this should be ready for merging in now.